clients/web/src/views/App.js

```diff
@@ -221,7 +223,8 @@ var App = View.extend({
 
             settings = _.extend(settings, {
                 el: this.$('#g-app-body-container'),
-                parentView: this
+                parentView: this,
+                brandName: this.brandName
```

Comment:
I think I will discuss this with @ronichoudhury because he wants to add some other configurations to this PR